### PR TITLE
dynomite 0.6.22 (new formula)

### PR DESCRIPTION
**Pull request by ankane**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

Dynamo layer for Redis and Memcached

### CODE REVIEW

**Review by chenrui333 — DISMISSED** [review]
LGTM

File: Formula/dynomite.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+  desc "Generic dynamo implementation for different k-v storage engines"
+  homepage "https://github.com/Netflix/dynomite"
+  url "https://github.com/Netflix/dynomite/archive/v0.6.22.tar.gz"
+  sha256 "9c3c60d95b39939f3ce596776febe8aa00ae8614ba85aa767e74d41e302e704a"
+  license "Apache-2.0"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "openssl@1.1"
```
Comment (carlocab) [inline Formula/dynomite.rb:11]: Doesn't work with OpenSSL 3?

File: Formula/dynomite.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+  def install
+    system "autoreconf", "-fvi"
+    system "./configure", *std_configure_args,
+                          "--disable-silent-rules",
+                          "--sbindir=#{bin}"
```
Comment (carlocab) [inline Formula/dynomite.rb:17]: Why not `sbin`?

File: Formula/dynomite.rb (hunk 1 above)
Comment (ankane) [reply Formula/dynomite.rb]: Looks like it does - updated

File: Formula/dynomite.rb (hunk 2 above)
Comment (ankane) [reply Formula/dynomite.rb]: Updated, but saw a few other formula used this pattern:

```text
grep -r '"--sbindir=#{bin}"' Formula
Formula/fping.rb:    system "./configure", *std_configure_args, "--sbindir=#{bin}"
Formula/cracklib.rb:                          "--sbindir=#{bin}",
Formula/smartmontools.rb:                          "--sbindir=#{bin}",
Formula/bacula-fd.rb:                          "--sbindir=#{bin}",
Formula/burp.rb:                          "--sbindir=#{bin}",
Formula/gnupg@2.2.rb:                          "--sbindir=#{bin}",
Formula/gnupg.rb:                          "--sbindir=#{bin}",
Formula/vnstat.rb:                          "--sbindir=#{bin}",
Formula/httpd.rb:                          "--sbindir=#{bin}",
Formula/ganglia.rb:                          "--sbindir=#{bin}",
```

Comment (carlocab) [reply Formula/dynomite.rb]: Yea, those are all really old formulae, so I suspect those flags are just from when Homebrew didn't handle `sbin` well.

**Review by SMillerDev — APPROVED** [review]

**Comment by SMillerDev** [comment]
Thanks @ankane  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Comment by ankane** [comment]
Thanks everyone!

**Pull request closed without merging**